.github/workflows/maven.yml

```diff
@@ -0,0 +1,22 @@
+name: Maven Build + Verify
+
+on: [ push ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
Maybe add a spotless check step here as well.

Reply:
Good catch. Added it.